assets/templates/partials/inputs/input-radio.tmpl

```diff
@@ -0,0 +1,13 @@
+<input
+  type="radio"
+  id="{{ .Id }}"
+  name="{{ .Name }}"
+  class="ons-radio__input ons-js-radio"
+  value="{{ .Value }}"
+  {{ if .Checked }}
+    checked
+  {{ end }}
+>  
+{{ if .LocaleKey }}
+  {{ template "partials/label" .}}
```

Comment:
In the wider scheme of things in dp-renderer, calling this partial "label" could be misunderstood as a label to be associated for any kind of element, when it is in fact only to be used with radio or checkbox inputs.

It really needs a more specific name.

This also applies to the reference in `input-checkbox.tmpl`

Reply:
In this case i think is better to move the label into their different input and conditional wrap count. Regarding  naming. what do you mean?